doc/source/index.rst

```diff
@@ -3,30 +3,69 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+========================================
 Welcome to stistool's documentation!
 ========================================
-This package provides data processing tools for working with STIS data.
+Stistools is a package that provides Python-based data processing tools for working with Space Telescope Imaging Spectrograph
+(STIS) data. It contains the full STIS calibration pipeline as well as its individual components should the user wish to do their
+calibrations manually. Additionally, stistools features a selection of analysis tools independent from the pipeline. These analysis
+tools are in active development, so there is more to come.
+
+.. note::
+
+    The information here reflect the *latest* software info and might not
+    be in-sync with the
+    `STIS Data Handbook <http://www.stsci.edu/hst/stis/documents/handbooks/currentDHB/>`_.
+
+Relationship to the IRAF/PyRAF stis Package
+*******************************************
+
+The intent of this package is to provide the user community with many of the useful tools contained within the original IRAF/PyRAF
+stis package in Python. This is consistent with a larger effort to transition the user community from IRAF/PyRAF to Python,
+which can be read about `here <https://stak-notebooks.readthedocs.io/en/latest/index.html>`_. At this time, there remains several tools
```

Comment:
When this goes in, let's open an issue to add a better/additional link here once I'm able to post a knowledge base article about the end of IRAF support.